### PR TITLE
[ch #160506498] Add logout button

### PR DESCRIPTION
**Pull request by Tittoh**
#### What does this PR do?
- Adds logout button
- Adds default image if user has not added profile image

#### Description of Task to be completed?
- The user will be able to logout

#### How should this be manually tested?
1. Clone this repo and branch git clone `git@github.com:andela/ah-jarvis-frontend.git -b ch-add-logout-button-160506498`
2. Open the project in your prefered text editor
3. Run the code below in your terminal
```
yarn install

yarn start
```
You should be redirected to your browser
4. Navigate to this url: `http://localhost:3000/signin`
5. Login
6. Click logout from the dropdown on the top right.

#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?

[#160506498 ](https://www.pivotaltracker.com/story/show/160506498) Add a logout route and link on the dashboard

#### Screenshots (if appropriate)
screen shot 2018-09-14 at 13 21 44

### CODE REVIEW

File: src/components/Header/index.js [hunk 1]
```diff
@@ -42,6 +46,10 @@ class Header extends React.Component {
             <li>
               <a href="#!">Favorites</a>
             </li>
+            <li className="divider" tabIndex="-1" />
+            <li>
+              <a href="#!" onClick={this.logout}>Logout</a>
```
Comment (yeenab) [inline src/components/Header/index.js]: @Tittoh Shouldn't the page be directed to the homepage after a successful logout

**Review by yeenab — CHANGES_REQUESTED** [review]

File: src/components/Header/index.js (hunk 1 above)
Comment (Tittoh) [reply src/components/Header/index.js]: Fixed it

File: src/components/Header/index.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ class Header extends React.Component {
 
   logout = () => {
     localStorage.removeItem('user');
+    return window.location.replace(ROUTES.home);
```
Comment (yeenab) [inline src/components/Header/index.js]: @Tittoh Let's not use window just a normal `Link` from *react-router-dom* is enough. Sth like
`<Link to 'ROUTES.home' />`

**Review by yeenab — CHANGES_REQUESTED** [review]
### NitPick

File: src/components/Header/index.js (hunk 2 above)
Comment (Tittoh) [reply src/components/Header/index.js]: This method updated the header only without redirecting, so I passed props and pushed history.

**Review by yeenab — APPROVED** [review]
### LGTM

**Pull request merged**